AnnoyancesFilter/sections/popups.txt

```diff
@@ -91,6 +91,7 @@ sodapdf.com##.be-tooltip
 !
 nejm.org#$#.featherlight { display: none !important; }
 nejm.org#$#html { overflow: visible !important; }
+blutv.com##.ant-btn
```

Comment:
Is it still required? Cannot find.

Reply:
"Register now"
<details>
<summary>Screenshot</summary>
<img src="https://user-images.githubusercontent.com/18536328/82452414-e206a380-9ab7-11ea-9302-d76417535fbe.png">
</details>

Reply:
This rule breaks register form.

Reply:
Ok, I see. So it can not be blocked safely on iOS.